storage/gcloud/aio/storage/bucket.py

```diff
@@ -47,10 +47,20 @@ def __init__(self, storage, name: str) -> None:
     async def list_blobs(self, prefix: str = '',
                          session: Optional[Session] = None) -> List[str]:
         params = {'prefix': prefix}
-        content = await self.storage.list_objects(self.name, params=params,
+        content = await self.storage.list_objects(self.name,
```

Comment:
I find that whenever you find yourself writing a loop like this where the first iteration needs to be treated specially, it's often the case that restructuring your code to avoid the duplication can lead to a much cleaner result. In this case, there's a couple potential cleanups you can make which will come together into a nice overall result:
* passing a param set to an empty string will act as if you hadn't sent that param at all. So on your first iteration you could have `'pageToken': ''`, which means you don't have to worry about selective in passing in / modifying your params only on the second and future iterations
* you do some juggling to make sure `content['items']` contains the full list -- in this case, it's easier to just pull the data out of `content` and into a collector after each API request (which could also be something you could do for all iterations rather than just the ones after the first!)

Pulling that all together, you could end up with something like this:
```python
params = {'prefix': prefix, 'pageToken': ''}
items = []
while True:
    content = await self.storage.list_objects(self.name,
                                              params=params,
                                              session=session)
    items.extend([x['name'] for x in content.get('items', list())])

    params['pageToken'] = content.get('nextPageToken')
    if not params['pageToken']:
        break

return items
```

Reply:
This makes a lot of sense! It looks much nicer than what I had written. Thank you for taking the time to give all of this feedback!